src/workspace/mod.rs

```diff
@@ -209,17 +213,74 @@ impl Workspace {
         }
     }
 
+    fn supports_soft_kill(&self, winid: u32) -> bool {
```

Comment:
is_softkill_supported ?:)
I don't know if this is best practice in rust, but in java ;) you prefix functions that return a boolean with is :)

Reply:
well lets do some java stuff 😂